configs/eval_internLM.py

```diff
@@ -0,0 +1,21 @@
+from mmengine.config import read_base
+
+from opencompass.models import internLM
+
+with read_base():
+    from .datasets.nq.nq_gen import nq_datasets
+
+datasets = [*nq_datasets]
```

Comment:
```suggestion
    from .datasets.collections.base_medium import datasets
```
Like how it was done in 
https://github.com/InternLM/opencompass/blob/main/configs/eval_internlm_7b.py



datasets = [*nq_datasets]
```